app/scripts/lib/createRPCMethodTrackingMiddleware.js

```diff
@@ -191,17 +162,44 @@ export default function createRPCMethodTrackingMiddleware({
 
       const properties = {};
 
+      let msgParams;
+
       if (event === EVENT_NAMES.SIGNATURE_REQUESTED) {
         properties.signature_type = method;
+
+        const data = req?.params?.[0];
+        const from = req?.params?.[1];
+        const paramsExamplePassword = req?.params?.[2];
+
+        const { isSIWEMessage } = detectSIWE({ data });
```

Comment:
I don't think we need this call here, it should only work with personal-sign. Why do we need the isSIWEMessage?

Reply:
Oh, I thought it's not just for the personal-sign. I'll fix it now.

Reply:
@brad-decker
You were right, there is no need for the `isSIWEMessage`... I don't know why I put it, thought it might was mandatory for all signatures, because at that moment I was looking into the request data being sent for the personal-sign. Suppose that was the case.
Thanks once again! Hope it's okay now.